js/base-util.js

```diff
@@ -49,33 +50,33 @@ function baseUtil () {
 		});
 	};
 
-	d20plus.ut.checkVersion = () => {
-		d20plus.ut.log("Checking current version");
-
-		function cmpVersions (a, b) {
-			const regExStrip0 = /(\.0+)+$/;
-			const segmentsA = a.replace(regExStrip0, "").split(".");
-			const segmentsB = b.replace(regExStrip0, "").split(".");
-			const l = Math.min(segmentsA.length, segmentsB.length);
-
-			for (let i = 0; i < l; i++) {
-				const diff = parseInt(segmentsA[i], 10) - parseInt(segmentsB[i], 10);
-				if (diff) {
-					return diff;
-				}
+	d20plus.ut.injectCode = (object, method, injectedCode, bindAndRun) => {
```

Comment:
am I right in thinking this is only used in:
```
		d20plus.ut.injectCode(d20.textchat, "incoming", d20plus.chat.r20incoming, true);
		d20plus.ut.injectCode(d20.textchat, "doChatInput", d20plus.chat.r20outgoing);
```
?

Could we not therefore do:

```
	d20plus.chat.r20outgoing = (originalDoChatInput, params) => {
		if (!params[2]) {
			d20plus.chat.resetSendMyself();
		}
		return originalDoChatInput(...params)
	}
```

and remove this extra `bindAndRun` arg/conditional branch?